packages/client/src/lib/intercom.js

```diff
@@ -0,0 +1,41 @@
+function intercom(name, email) {
+  (function () {
+    var w = window;
+    var ic = w.Intercom;
+    if (typeof ic === 'function') {
+      ic('reattach_activator');
+      ic('update', w.intercomSettings);
+    } else {
+      var d = document;
+      var i = function () {
+        i.c(arguments);
+      };
+      i.q = [];
+      i.c = function (args) {
+        i.q.push(args);
+      };
+      w.Intercom = i;
+      var l = function () {
+        var s = d.createElement('script');
+        s.type = 'text/javascript';
+        s.async = true;
+        s.src = 'https://widget.intercom.io/widget/gx67sx4f';
+        var x = d.getElementsByTagName('script')[0];
+        x.parentNode.insertBefore(s, x);
+      };
+      l();
+    }
+  })();
+
+  var options = {
+    app_id: 'gx67sx4f',
+    name: name,
+    email: email,
+    created_at: Math.floor(Date.now() / 1000),
```

Comment:
I'm quite sure created_at is supposed to be when the user was created, not the current date.

Reply:
Yes, will fix.